Modules/cjkcodecs/cjkcodecs.h

```diff
@@ -60,11 +60,17 @@ struct pair_encodemap {
     DBCHAR code;
 };
 
-typedef struct {
+#ifndef CJK_MOD_SPECIFIC_STATE
+#define CJK_MOD_SPECIFIC_STATE
+#endif
+
+typedef struct _cjk_mod_state {
     int num_mappings;
     int num_codecs;
     struct dbcs_map *mapping_list;
     MultibyteCodec *codec_list;
+
+    CJK_MOD_SPECIFIC_STATE
```

Comment:
IIUC, can we always reserve the fields for big5_encmap / big5_decmap ?
I am not sure upsides and downsides of both approaches.

Reply:
But we also need room for extra module state in the iso2022 codec.

Reply:
Ah, okay, now I understood the approach.. this is super nice.